web/src/api/images/delete_all_images.ts

```diff
@@ -0,0 +1,159 @@
+import {
+  errorHasuraQuery,
+  errorNotAllowed,
+  errorResponse,
+} from "src/backend/errors";
+import { NextApiRequest, NextApiResponse } from "next";
+import { protectInternalEndpoint } from "src/backend/utils";
+import { getAPIServiceGraphqlClient } from "src/backend/graphql";
+import { getSdk as getUnverifiedImagesSDK } from "@/api/images/graphql/getUnverifiedImages.generated";
+import {
+  S3Client,
+  ListObjectsCommand,
+  PutObjectTaggingCommand,
+} from "@aws-sdk/client-s3";
+import { logger } from "@/lib/logger";
+
+export type DeleteAllImagesResponse = {
+  success?: boolean;
+};
+
+/**
```

Comment:
Think this makes sense to implement the actual trigger in a seperate pr from the image upload process. Just leaving the endpoint here for now.